services/api/src/models/definitions/user.json

```diff
@@ -48,98 +48,93 @@
       "type": "Boolean",
       "default": false
     },
-    "hashedPassword": {
-      "type": "String",
-      "readScopes": "none",
-      "writeScopes": "none"
-    },
-    "loginAttempts": {
-      "type": "Number",
-      "default": 0,
-      "readScopes": "none",
-      "writeScopes": "none"
-    },
-    "lastLoginAttemptAt": {
-      "type": "Date",
-      "readScopes": "none",
-      "writeScopes": "none"
-    },
     "accessConfirmedAt": {
       "type": "Date",
-      "writeScopes": "none"
+      "writeAccess": "none"
     },
-    "authInfo": [
-      {
-        "lastUsedAt": {
-          "type": "Date",
-          "required": true,
-          "writeScopes": "none"
+    "$private": {
+      "type": "Scope",
+      "readAccess": "none",
+      "writeAccess": "none",
```

Comment:
should these be removed if all the fields are private ?

Reply:
field are always public by default so if they're fully private then this belongs here